apps/io-wallet-user-func/src/email-notification-service.ts

```diff
@@ -0,0 +1,22 @@
+import { MailerConfig } from "@pagopa/io-functions-commons/dist/src/mailer";
+import * as TE from "fp-ts/lib/TaskEither";
+import * as t from "io-ts";
+
+export const SendEmailParams = t.type({
+  html: t.string,
+  subject: t.string,
+  text: t.string,
+  to: t.string,
+});
+
+export type SendEmailNotificationParams = t.TypeOf<typeof SendEmailParams>;
+
+export interface IEmailNotificationService {
+  getTransporter: (
+    transporter: MailerConfig,
+    params: SendEmailNotificationParams,
+  ) => MailerConfig;
```

Comment:
If `getTransporter` is never used by the user but is just an internal method for `sendEmail` it should not be exported and I would not include it in the interface.